warlog.js

```diff
@@ -97,6 +104,20 @@ exports.handler = async (event, context) => {
     return results;
 };
 
+const equalDeck = (warDeck, otherDecks) => {
```

Comment:
I don't understand why this is diffing, but I figure it out

Reply:
I'm not sure, I downloaded the current master branch when I worked on the environment variables. Let me know if you figure something out :)